app/controllers/idv/image_uploads_controller.rb

```diff
@@ -52,8 +52,8 @@ def render_form_response(form_response)
           Array(errs).map { |err| { field: key, message: err } }
         end
 
-        render json: form_response.to_h.merge(errors: errors),
-               status: :bad_request
+        render json: form_response.to_h.except(:status).merge(errors: errors),
```

Comment:
I don't like this `.except(:status)` but without adding it all the way up  to `FormResponse` or `DocAuth::Response` I guess there's not a good way :/

Reply:
In earlier iterations I'd tried to make it so that this would call to the form's `status` method directly, only later realizing that we have the submission _response_ here, which may also be the response from the doc auth client. Maybe there's a way we can make the `ApiImageUploadForm` instance available here?

Reply:
I guess since know that the only form response that has a status is the image form one... maybe we could do something like


```
else
  render_form_response(form_response, status: image_form.status)
end
```

Reply:
> I guess since know that the only form response that has a status is the image form one... maybe we could do something like
> 
> ```
> else
>   render_form_response(form_response, status: image_form.status)
> end
> ```

Yeah, that approach seems to work pretty well. Updated in 45c3f06.